extension/sql/vectorize--0.13.0--0.14.0.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE vectorize.job ADD COLUMN index_dist_type TEXT NOT NULL DEFAULT 'pgv_hsnw_cosin';
+
```

Comment:
since we are changing the signature of the functions (in api.rs), we need to either drop and re-add those functions, or define them in the migration as well. The code in meta.sql is not executed for any user updating via `alter extension update`.

Here's an example of where this has been done: https://github.com/tembo-io/pg_vectorize/blob/550946c61e28f0b67f72df51798b51618f393b87/extension/sql/vectorize--0.8.1--0.9.0.sql#L3-L19